moped-editor/src/routes.js

```diff
@@ -20,20 +20,20 @@ import DashboardView from "./views/dashboard/DashboardView";
 export const routes = [
   { path: "/", element: <Navigate to="/moped" /> },
   {
-    path: "moped/session",
+    path: "/moped/session",
     element: <MainLayout />,
     children: [
       { path: "signin", element: <LoginView /> },
       { path: "*", element: <Navigate to="/signin" /> },
     ],
   },
   {
-    path: "moped",
+    path: "/moped",
     action: "moped:visit",
     element: <DashboardLayout />,
     children: [
       {
-        path: "/",
```

Comment:
react-router now throws an error if you try to set the root `/` as a child. tbh i'm not sure why this was working in the first place :/